lib/msal-browser/README.md

```diff
@@ -10,12 +10,46 @@ The MSAL library for JavaScript enables client-side JavaScript applications to a
 [![Build Status](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-js.png?branch=dev)](https://travis-ci.org/AzureAD/microsoft-authentication-library-for-js)[![npm version](https://img.shields.io/npm/v/msal.svg?style=flat)](https://www.npmjs.com/package/msal)[![npm version](https://img.shields.io/npm/dm/msal.svg)](https://nodei.co/npm/msal/)
 
 ## Installation
-### Via NPM:
+### Via NPM (Not available yet):
 
     npm install msal-browser
 
-## How to Run Samples for Authorization Code Flow POC
-See the PR [here](#1106).
+## Build and Run
+
+### Build Library
+```javascript
+// To run build only for browser package
+npm run build
+// To run build for common and browser package
+npm run build:all
+```
+
+### Run
+To run sample, ensure the client_id and client_secret are updated and filled in.
```

Comment:
We need to make it very clear that this is temporary (should not be provided once portal has been updated) and should not be deployed to production environments.

Reply:
Done, changed the name of the variable, added logger warnings, added comments in the readme.